### DISCUSSION TITLE
v2.10.0

### DISCUSSION THREAD

**Discussion by mika-f** [mika-f/blender-drag-and-drop#63] [Announcements]
## What's Changed
* Add support for Blender 3.6.4 by @Jyrka98 in https://github.com/mika-f/blender-drag-and-drop/pull/61
* Release v2.10.0 by @mika-f in https://github.com/mika-f/blender-drag-and-drop/pull/62

## New Contributors
* @Jyrka98 made their first contribution in https://github.com/mika-f/blender-drag-and-drop/pull/61

**Full Changelog**: https://github.com/mika-f/blender-drag-and-drop/compare/v2.9.0...v2.10.0

## NOTES

* Restart Blender after updating add-on. Due to limitations of Windows, changes will not take effect until restart.

<hr /><em>This discussion was created from the release <a href='https://github.com/mika-f/blender-drag-and-drop/releases/tag/v2.10.0'>v2.10.0</a>.</em>

**Comment by dkpress** [comment]
thank you very mush!! ❤
